bokeh/models/formatters.py

```diff
@@ -286,6 +286,10 @@ class LogTickFormatter(TickFormatter):
     base to use. If unset, the formatter will use base 10 as a default.
     """)
 
+    min_exponent = Int(0, help="""
+    Minimum exponent to format in scientific notation.
```

Comment:
I think this needs a little more elaboration to make it clear that this applies to the *absolute value* of any exponent (i.e it applies to large negative exponents too)